### ISSUE TITLE
Fatal error with last PR

### ISSUE THREAD

**Issue by annando** [friendica/friendica#4346]
```
[27-Jan-2018 23:32:49 UTC] PHP Warning:  Imagick::readimageblob() expects parameter 1 to be string, object given in /.../src/Object/Image.php on line 135
[27-Jan-2018 23:32:49 UTC] PHP Fatal error:  Uncaught ImagickException: Can not process empty Imagick object in /.../src/Object/Image.php:149
```

**Comment by MrPetovan** [comment]
Do you have any more context?

**Comment by MrPetovan** [comment]
Never, I found the problem in `src/Content/Text/BBCode.php:523`, we feed the return of `OEmbed::fetchUrl()` which is an object to `Image::__construct` which expects a string. It is supposed to be `Network::fetchUrl()` instead.

**Comment by annando** [comment]
Okay. Du you create the PR?

**Comment by MrPetovan** [comment]
Fixed by #4347
